cordova-lib/src/plugman/plugman.js

```diff
@@ -169,35 +160,6 @@ plugman.commands =  {
             }
         });
     },
-
-    'publish'  : function(cli_opts) {
```

Comment:
+1 to leaving the function stubs around for a bit and simply have them output that this functionality is not supported anymore.